CHANGELOG.md

```diff
@@ -1,72 +1,177 @@
+<a name="unreleased"></a>
+# [Unreleased]
+
+<a name="2.2.0"></a>
+# [2.2.0] - 2024-11-04
+
+## CXoneChatSDK
```

Comment:
It is kinda strange that we have one changelog for multiple projects

Reply:
Yeah good point, I noticed that after I've released the SDK. I'm going to clean the changelog up based on the features related to the release starting with next release. Let's keep it for this one and handle it correctly, since iOS has separate repository for each module, for future releases.